### ISSUE TITLE
bug - Workload (splk-wlk) - In some circumstances the Splunk scheduler logs can lack app and user context leading to the creation of new entities in case of execution errors detected

### ISSUE THREAD

**Issue by sebwurl** [trackme-limited/trackme-report-issues#195]
Hi,
saved searches that are changed to SPL with errors, will be created as new entities by the tracker because the scheduler log does not provide a user context. The new entity for the saved search will be created for user "system". Entity syntax is like <app>:system:<search_name>.

**Comment by guilhemmarchand** [comment]
Thanks @sebwurl 

I agree and saw this too, this is a little it inconsistent logging from Splunk (a bug IMO), the sheduler usually has app and user context but it's true in some circumstances it is missing, leading to these consequences for TrackMe given that we need to take into account the user and app context.

Working on the validation of a workaround, wil update you shortly

**Comment by guilhemmarchand** [comment]
If you have a chance @sebwurl to check in your lab, the build 1689144916 addresses the matter:
https://github.com/trackme-limited/trackme-report-issues/releases/tag/beta-version-2.0.43

**Comment by sebwurl** [comment]
Okay :) So I deleted and re-created the scheduler tracker.
For a search I created afterwards, it looks fine.

Searches that existed before were adding more and more entities due to duplicating the user info:
I re-created the tracker at 14:59. Afterwards a new entity was created every 5min (with every tracker run)
![image](https://github.com/trackme-limited/trackme-report-issues/assets/85566586/6e728adf-826e-45f3-b29d-bdbb6fa66e23)

**Comment by guilhemmarchand** [comment]
```
For a search I created afterwards, it looks fine.
```
Yes I could validate it too ;-)

```
Searches with errors that existed before were adding more and more entities due to duplicating the user info:
I re-created the tracker at 14:59. Afterwards a new entity was created every 5min (with every tracker run)
```

This is to be expected beause the main tracker looks at metrics generated by the children trackers, like the scheduler, so previously generated metrics remain valid unless you destroy the tenant and create another one with a different name - or wait 24h

**Comment by guilhemmarchand** [comment]
Addressed in 2.0.43 @sebwurl
